torch/_subclasses/fake_tensor.py

```diff
@@ -1517,7 +1517,7 @@ def _cached_dispatch_impl(
         output = self._dispatch_impl(func, types, args, kwargs)
 
         try:
-            self._validate_cache_key(func, args, kwargs)
+            entry = self._make_cache_entry(state, key, func, args, kwargs, output)
```

Comment:
This is just clean-up. Originally we validated the cache key before running the operation but now that we compute the output first there's no reason for this to be a separate check from the _make_cache_entry() below.